### PR TITLE
[DRAFT] feat/DES 36 radio button

### PR DESCRIPTION
**Pull request by hcvdhaar**
Waiting for final design to be ready. Maybe some stuff needs to be updated based on that.

- feat: radio button first setup
- feat: addding disabled state to storybook
- feat: use amsterdam tokens
- docs: added readme
- feat: using private variables to set state updates
- docs: prepare for new docs format

### CODE REVIEW

File: packages/css/src/radio-button/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Radio button
+
+We gebruiken voor de `RadioButton` de community css en het React component.
```
Comment (alimpens) [inline packages/css/src/radio-button/README.md:3]: Maybe we can explain a bit better what the 'community' is, for people not familiar with NLDS?

File: storybook/storybook-react/src/components/RadioButton/RadioButton.docs.mdx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+<Markdown>{README}</Markdown>
+
+<Primary />
+
+<Controls />
+
+## Stories
+
+De primary, secondary en tertiary buttons kunnen naast elkaar gebruikt worden. Niveaus mogen worden overgeslagen.
```
Comment (alimpens) [inline storybook/storybook-react/src/components/RadioButton/RadioButton.docs.mdx:15]: Is this relevant for the `RadioButton`?

File: packages/css/src/radio-button/README.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Radio button
```
Comment (alimpens) [inline packages/css/src/radio-button/README.md:1]: General question: do we want to offer a radio button like this, without a label? You would never use it without a label right?

File: packages/css/src/radio-button/radio-button.scss [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+// Similar concept as Utrecht to set private variables and update them when specific state is set.
+.amsterdam-radio-button {
+  --_amsterdam-radio-button-background-color: var(--amsterdam-radio-button-background-color, currentColor);
+}
+
+.amsterdam-radio-button:checked {
+  background-image: radial-gradient(
+    circle,
+    var(--_amsterdam-radio-button-background-color, transparent) (55% - 5%),
+    currentColor 60%
```
Comment (alimpens) [inline packages/css/src/radio-button/radio-button.scss:12]: I think the styling is off,  a checked radio should have a blue border, then a white border, then a blue fill. It's completely filled in the current implementation (on my machine at least)

File: packages/css/src/radio-button/radio-button.scss [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  );
+}
+
+.amsterdam-radio-button:hover {
+  box-shadow: var(--amsterdam-radio-button-hover-box-shadow);
```
Comment (alimpens) [inline packages/css/src/radio-button/radio-button.scss:17]: The hover `box-shadow` should be inset, I think. The shadow is currently visible for disabled buttons as well, that shouldn't be the case I think?

File: storybook/storybook-react/src/components/RadioButton/RadioButton.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+const meta = {
+  title: 'Radio Button',
+  component: RadioButton,
+  args: {
+    checked: false,
+    invalid: false,
+    disabled: false,
+    onChange: () => {
+      console.log('onChange')
+    },
```
Comment (alimpens) [inline storybook/storybook-react/src/components/RadioButton/RadioButton.stories.tsx:20]: I don't know it this is very useful. Currently, it logs an entry when you click on it, but the state of the radio button doesn't actually change

**Comment by hcvdhaar** [comment]
For now closing this PR. Story will be updated with more specs

**Pull request closed without merging**